pkg/controller/embercsi/embercsi_controller.go

```diff
@@ -536,6 +529,20 @@ func getVolumeMounts(ecsi *embercsiv1alpha1.EmberCSI, csiDriverMode string) []co
 		},
 	}
 
+        // Check to see if the volume driver is LVM
+        if len(ecsi.Spec.Config.EnvVars.X_CSI_BACKEND_CONFIG) > 0 && strings.Contains(ecsi.Spec.Config.EnvVars.X_CSI_BACKEND_CONFIG, "cinder.volume.drivers.lvm.LVMVolumeDriver")  {
```

Comment:
Shouldn't we have failed somewhere else if `X_CSI_BACKEND_CONFIG` was empty?  If we don't have that check we need to add it.

I think we should deserialize the `X_CSI_BACKEND_CONFIG` parameter and store it for the duration of the operation so we can easily access its elements, in this case the driver option.

Reply:
I am going to do that for another issue #5. This will still be relevant thought and is a cheap way to identify the LVM backend without unmarshelling the json file everytime.

Reply:
s/file/string/